tracer/resources/args_parser.py

```diff
@@ -16,7 +16,7 @@
 # 02110-1301, USA.
 #
 
-import argparse
+import argparse, argcomplete
```

Comment:
Okay, that makes more sense.. I wish you said that to start with! And I don't want you to have to apply patches every time you release. I wasn't tying to be obstructive, just trying to understand the use case.

I'll change the hard dependency to a soft one tomorrow :+1:  For arch maybe an `optdepends` would make sense